sql/driver/driver_test.go

```diff
@@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
 	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
```

Comment:
This addition doesn't seem to be testing much. I'd probably just remove it for now. You're correct that our testing is fairly rudimentary at the moment.